docs/registering-ticket-sales.md

```diff
@@ -46,58 +46,66 @@ Your application then starts its typical flow of guiding the passholder through
 
 At some point during the checkout process on your website or application (but **before a payment has happened**), you provide the passholder a way to enter their UiTPAS number if they have one.
 
-### 4. Determine possible UiTPAS tariffs
+### 4. Determine available UiTPAS tariffs
 
-Using the event id, the UiTPAS number and the regular price of your event, you can [request possible UiTPAS tariffs](/reference/UiTPAS.v2.json/paths/~1tariffs/get).
+Using the event id, the UiTPAS number and the regular price of your event, you can [request available UiTPAS tariffs](/reference/UiTPAS.v2.json/paths/~1tariffs/get).
 
 Example request:
 
 ```http
-GET /tariffs/?eventId=EVENT_ID&uitpasNumber=UITPAS_NUMBER&regularPrice=10 HTTP/1.1
+GET /tariffs/?eventId=5a0967f9-cc06-4c3c-9206-30481a767434&uitpasNumber=0900000672312&regularPrice=10 HTTP/1.1
```

Comment:
I'm assuming that this is test data from the test environment. If that's correct, then we should probably also change the `Host: https://api.uitpas.be` underneath this line to `Host: api-test.uitpas.be` (the `https://` should also not be included in the Host header, probably a mistake I made earlier)

Reply:
I see that it's also the production URL in the other guides